packages/docs/site/docs/developers/05-local-development/04-wp-playground-cli.md

```diff
@@ -0,0 +1,118 @@
+
+``bash
+cd my-plugin-or-theme-directory
+npx @wp-playground/cli@latest server --auto-mount
+
+````
+
+### Choosing a WordPress Version
+
+By default, the CLI loads the latest stable version of WordPress and PHP 8.0 due to its improved performance. To specify your preferred versions, you can use the flag `--wp=<version>` and `--php=<version>`:
+
+```bash
+npx @wp-playground/cli@latest server --wp=6.8 --php=8.4
+````
+
+### Mounting folders manually
+
+Some projects have a specific structure that requires a custom configuration, for example, your repo contains all the files from the `/wp-content/` folder. So this scenario you can specify to the Plaground CLI you will mount your project from that folder using the flag `--mount`.
+
+```bash
+npx @wp-playground/cli@latest server --mount=.:/wordpress/wp-content/plugins/
+```
+
+### Loading Blueprints
+
+A way to bring the CLI playground to the next level is to integrate with Blueprints, which allows developers to set up the initial state from their WordPress instance. With the flag `--blueprint` the developer will be capable run a Playground with a custom inital state.
+
+**(my-blueprint.json)**
+
+```bash
+{
+  "landingPage": "/wp-admin/options-general.php?page=akismet-key-config",
+  "steps": [
+    {
+      "step": "installPlugin",
+      "pluginData": {
+        "resource": "wordpress.org/plugins",
+        "slug": "akismet"
+      },
+      "options": {
+        "activate": true
+      }
+    },
+    {
+      "step": "login",
+      "username": "admin",
+      "password": "password"
+    },
+  ]
+}
+```
+
+```bash
+npx @wp-playground/cli@latest server --blueprint=my-blueprint.json
+```
+
+## Command and Arguments
+
+Playground CLI is simple, configurable, and unopinionated. You can set it up according
+to your unique WordPress setup. With the Playground CLI, you can use the following top-level commands:
+
+-   **`server`**: (Default) Starts a local WordPress server.
+-   **`run-blueprint`**: Executes a Blueprint file without starting a web server.
+-   **`build-snapshot`**: Builds a ZIP snapshot of a WordPress site based on a Blueprint.
+
+The `server` command supports the following optional arguments:
+
+-   `--port=<port>`: The port number for the server to listen on. Defaults to 9400.
+-   `--outfile`: When building, write to this output file.
+-   `--wp=<version>`: The version of WordPress to use. Defaults to the latest.
+-   `--auto-mount`: Automatically mount the current directory (plugin, theme, wp-content, etc.).
+-   `--mount=<mapping>`: Manually mount a directory (can be used multiple times). Format: /host/path:/vfs/path
+-   `--mount-before-install`: Mount a directory to the PHP runtime before WordPress installation (can be used multiple times). Format: `"/host/path:/vfs/path"`.
```

Comment:
Let's mention why that matters (when you want to use your local WordPress files or install WordPress in a local directory and access its files later). I'm thinking of renaming these options to `--mount-before-install` and `--mount-after-install`